src/gimelstudio/core/node/property.py

```diff
@@ -144,6 +145,29 @@ def CreateUI(self, parent, sizer):
         self.AddToFoldPanel(sizer, fold_panel, self.img)
 
 
+class ColorProp(Property):
+    """ 
+    Allows the user to select a color.
+    """
+    def __init__(self, idname, default=(255, 255, 255), visible=True):
+        Property.__init__(self, idname, default, fpb_label, expanded, visible)
+    
+    def CreateUI(self, parent, sizer):
+        fold_panel = self.CreateFoldPanel(sizer)
+        fold_panel.SetBackgroundColour(wx.Colour("#464646"))
```

Comment:
This line is actually not needed since ``self.CreateFoldPanel`` already sets the background color. 

(If you were using a ``wx.Panel`` to contain the widget like the ``XYZprop``, then you would want to set the background color, but not if you directly place it in the fold_panel.)